preface.qmd

```diff
@@ -7,7 +7,7 @@ We aim to *minimize surprises*, with each section and chapter building on the pr
 If you're just starting out with Python for working with geographic data, this book is an excellent place to start.
 
 There are many resources on Python on 'GeoPython' but none that fill this need for an introductory resource that provides strong foundations for future work.
-We want to avoid reinventing the wheel and provide something that fills an 'ecological niche' in the wider free and open source software for geospatial (FOSS4G) ecosystem.
+We want to avoid reinventing the wheel and provide something that fills an 'ecological niche' in the wider free and open-source software for geospatial (FOSS4G) ecosystem.
```

Comment:
Was going to question this but, having looked it up, I think this is right and good, in open-source software it's an adjective and that seems right: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source